acestep/training_v2/ui/summary.py

```
@@ -80,52 +119,104 @@ def _show_rich(
         Panel(
-            Group(table, hints),
+            Group(*parts),
             title="[bold green]Training Complete[/]",
             border_style="green",
             padding=(0, 1),
         )
     )
 
 
+def _build_checkpoint_grid(
+    stats: TrainingStats, output_dir: str,
```

Ruff (0.15.0)

[warning] Unused function argument: `output_dir`

(ARG001)